src/style.css

```diff
@@ -123,4 +121,83 @@ footer {
   text-align: center;
   background-color: #f5f4ef;
   border-top: 3px ridge #555;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+.createPopup {
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.8);
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  pointer-events: none;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  opacity: 0;
+  transform: translateY(100%);
+  transition: transform 0.9s;
+}
+
+.invisible {
+  pointer-events: auto;
+  opacity: 1;
+  transform: translateY(0%);
```

Comment:
I was going to say `display: none;` might be better, but no, you are right. This is definitely the better and prettier way to do it. It looks really awesome like this.